code/datums/martial/cqc.dm

```diff
@@ -213,9 +213,9 @@
 
 /mob/living/carbon/human/proc/CQC_help()
 	set name = "Remember The Basics"
-	set desc = "You try to remember some of the basics of CQC."
+	set desc = "You try to remember the basics of CQC."
 	set category = "CQC"
-	to_chat(usr, "<b><i>You try to remember some of the basics of CQC.</i></b>")
+	to_chat(usr, "<b><i>You try to remember the basics of CQC.</i></b>")
```

Comment:
The choice of words here is deliberate because it's a reference to a line from _Metal Gear Solid_